lib/ui/form/form_screen.dart

```diff
@@ -77,8 +77,8 @@ class FormScreenState extends ConsumerState<FormScreen> {
     SurveyDetailsModel? surveyDetails,
     String errorMessage = "",
   }) {
-    // TODO: Integrate item count from survey details #25
-    const questionTotal = 5;
+    final questions = surveyDetails?.questions ?? [];
+    final questionTotal = questions.length;
```

Comment:
I think we should move down these lines of code to below of `if (surveyDetails != null) ...[` to eliminate null check

Reply:
@luongvo I added the null-check because I didn't want to pass a nullable object to `FormSurveyDetailPage` (it's annoying to have to add default values again if it's null)

Thoughts? 🙏

Reply:
@Tuubz I thought we can move it down but actually can not. Sorry for the confusion 🙏